src/db/oracledb/books-dao.js

```diff
@@ -45,7 +45,7 @@ const getBookById = async (id) => {
   const query = `
     SELECT *
     FROM library_api_books
-    WHERE book_id = :bookId
+    WHERE bookid = :bookId
```

Comment:
Hmm, why is the column name changed? In addition, I don't think `bookid` is following any naming convention, so why do we want to change it from `book_id` to `bookid`?

Reply:
fixed.